src/tests/suite/daos_test_common.c

```diff
@@ -1017,3 +1019,124 @@ get_daos_prop_with_user_acl_perms(uint64_t perms)
 	D_FREE(user);
 	return prop;
 }
+
+/* JSON output handling for dmg command */
+static int
+daos_dmg_json_contents(const char *dmg_cmd, struct json_object **parsed_json)
+{
+	long	int  size = 0;
+	char	*content = NULL;
+	char	*filename = "/tmp/daos_dmg.json";
```

Comment:
may need to randomize this or something.  It won't work if someone leaves something around.